### ISSUE TITLE
Yagna log is flooded with payment driver errors

### ISSUE THREAD

**Issue by evik42** [golemfactory/yagna#2677]
My yagna spams these log messages every few millisecond:

```
[2023-07-28T17:04:35.054+0200 INFO  ya_erc20_driver::driver::cron] Transaction not sent, retrying
[2023-07-28T17:04:35.054+0200 WARN  ya_erc20_driver::driver::cron] Transaction not found on chain for None
[2023-07-28T17:04:35.054+0200 WARN  ya_erc20_driver::driver::cron] Time since last action Duration { secs: 298, nanos: 997039100 }
[2023-07-28T17:04:35.076+0200 ERROR ya_erc20_driver::driver::cron] Error when getting get_tx_on_chain_status: GenericError { inner: "Web3 clients failed." }
[2023-07-28T17:04:35.101+0200 ERROR ya_erc20_driver::driver::cron] Error when getting get_tx_on_chain_status: GenericError { inner: "Web3 clients failed." }
[2023-07-28T17:04:35.129+0200 ERROR ya_erc20_driver::driver::cron] Error when getting get_tx_on_chain_status: GenericError { inner: "Web3 clients failed." }
[2023-07-28T17:04:35.157+0200 ERROR ya_erc20_driver::driver::cron] Error when getting get_tx_on_chain_status: GenericError { inner: "Web3 clients failed." }
[2023-07-28T17:04:35.183+0200 ERROR ya_erc20_driver::driver::cron] Error when getting get_tx_on_chain_status: GenericError { inner: "Web3 clients failed." }
```

maybe the system should back off a little instead of flooding the logs and probably some web3 endpoint?

[logs.zip](https://github.com/golemfactory/yagna/files/12195865/logs.zip)

**Comment by nieznanysprawiciel** [comment]
`golemsp` runs `yagna payment init`
```
for nn in NETWORK_GROUP_MAP[&config.account.network].iter() {
        for driver in DRIVERS.iter() {
            if driver.platform(nn).is_err() {
                continue;
            }

            if let Err(e) = cmd.yagna()?.payment_init(&address, nn, driver).await {
                log::debug!("Failed to initialize {} driver. Error: {e}", driver.name);
            }
        }
    }
```
We have problem only on requestor side

**Comment by evik42** [comment]
@nieznanysprawiciel Why did you close this issue without understanding what it is about?

This happens after `yagna payment init --sender` and using the requestor for some requests then it goes into this flood loop.
Also these errors means that the IP is blocked on the web3 server because of the flood. I had to wait ~1 hour to start to work with a new address/new data dir because I got back 429 error codes.

Basically if a transaction fails for any reason, the payment driver goes into a flood loop because on error it retries again and again.

**Comment by scx1332** [comment]
Is this issue still valid?
